app/controllers/api/v1/students_controller.rb

```diff
@@ -4,4 +4,12 @@ def books
     @books = @student.books
     render json: BookSerializer.new(@books.paginate(params[:per_page], params[:page]))
   end
+
+  def bookmarks
+    @student = Student.find(params[:id])
+    @bookmarks = @student.student_books.map do |student_book|
```

Comment:
will @student be used? actually since there are no views, we don't need the @ right?

Reply:
You're right-- we don't need the @ symbol. Force of habit!